### PR TITLE
Update installation.markdown

### PR DESCRIPTION
**Pull request by NeLLyMerC**
## Type of change

- [x ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- This PR fixes or closes issue:

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -32,14 +32,14 @@ The following will take you through the steps required to install Hass.io.
...
+   - Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space. On Virtual machine platforms, provide at least 32 GB of disk space for the VM.
    - Load the appliance image into your virtual machine software. Choose 64-bit Linux and UEFI boot.
 
 3. Optional - set up the WiFi or static IP. There are two possible places for that:
    - on a blank USB stick with a FAT32 partition having partition label `CONFIG`, while in its root directory, create the `network/my-network` file, or
-   - on the Hass.io SD card's first, bootable partition (labeled `hassio-boot`, might not be auto mounted in Linux) create the `CONFIG/network/my-network` file.
+   - on the Home Assistant SD card's first, bootable partition (labeled `hassio-boot`, might not be auto mounted in Linux) create the `CONFIG/network/my-network` file.
 
    For the content of this file, follow the [HassOS howto][hassos-network].
```
Comment (frenck) [inline source/hassio/installation.markdown]: HassOS became the Home Assistant Operating System

File: source/hassio/installation.markdown [hunk 2]
```diff
@@ -67,39 +67,39 @@ If you are using a Raspberry Pi please remember to ensure you're using an [appro
...
+## Run the beta version on Home Assistant
 
 If you would like to test next release before anyone else, you can install the beta version released every three weeks:
 
-1. Backup your installation, using the snapshot functionality Hass.io offers.
+1. Backup your installation, using the snapshot functionality Home Assistant offers.
 2. Check the [Home Assistant RC release notes](https://rc.home-assistant.io/latest-release-notes/) for breaking changes. Be sure to check all release notes between the version you are running and the one you are upgrading to. Use the search function in your browser (`CTRL + f`) and search for **Breaking Changes**.
-3. Select _System_ tab from the _Hass.io_ menu, then select _Join Beta Channel_ under _Hass.io supervisor_, then select _Reload_.
-4. Select _Dashboard_ tab from the _Hass.io_ menu, and then select _Update_.
+3. Select _System_ tab from the _Home Assistant_ menu, then select _Join Beta Channel_ under _Home Assistant supervisor_, then select _Reload_.
```
Comment (frenck) [inline source/hassio/installation.markdown]:
```suggestion
3. Select _System_ tab from the _Supervisor_ menu, then select _Join Beta Channel_ under _Supervisor_, then select _Reload_.
```

File: source/hassio/installation.markdown [hunk 3]
```diff
@@ -67,39 +67,39 @@ If you are using a Raspberry Pi please remember to ensure you're using an [appro
...
+4. Select _Dashboard_ tab from the _Home Assistant_ menu, and then select _Update_.
```
Comment (frenck) [inline source/hassio/installation.markdown]:
```suggestion
4. Select _Dashboard_ tab from the _Supervisor_ menu, and then select _Update_.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by NeLLyMerC — COMMENTED** [review]
There may be an error in me fixing links. I dont have prior experience with markdown.

File: source/hassio/installation.markdown (hunk 1 above)
Comment (NeLLyMerC) [reply source/hassio/installation.markdown]: I think I did what you suggested? I haven’t done this before, so I’m learning through this first attempt. Thanks for your patience

**Review by frenck — APPROVED** [review]
Perfect, thanks! 👍

**Pull request merged**